iac-extensions/docker/src/test/java/org/sonar/iac/docker/reports/HadolintImporterTest.java

```diff
@@ -154,6 +172,37 @@ void unknownRule(String reportPath) {
     verifyNoInteractions(mockAnalysisWarnings);
   }
 
+  @ParameterizedTest
+  @CsvSource({
+    "src/test/resources/hadolint/jsonFormat/unknownWarningRuleWithValidHadolintName.json, SC9999",
```

Comment:
As `src/test/resources/hadolint/` is a repeating sequence you could use a constant for this and concat within the method.